docs/development_suite/identity-and-access-management/overview.md

```diff
@@ -1,11 +1,29 @@
 
-The Console is based on hierarchical resources: assigning specific roles and permissions to both Users and Service Accounts is important for defining responsibilities within your Companies and Projects. Discover more about it in the [Console Levels and Permission Management](/development_suite/identity-and-access-management/console-levels-and-permission-management.md) documentation.
+The Console is based on hierarchical resources: assigning specific roles and permissions to the above mentioned Identity types is important for defining responsibilities and privileges within your Companies and Projects. Discover more about it in the [Console Levels and Permission Management](/development_suite/identity-and-access-management/console-levels-and-permission-management.md) documentation.
 
-Users with enough administrative permission on a specific Company or Project can view the existing Users and Service Accounts and perform actions on them. Here is an example of the Identities table of a Company, which can be found in the IAM (Identity and Access Management) section:  
+## Identities Portal
 
-![identities table](./img/identities_table.png)
\ No newline at end of file
+Users with enough administrative permission can manage roles and permissions given to Users, Service Accounts and Groups and perform actions on them at both Company and Project levels. This grants them ownership over the decision-making process regarding which resources are accessible and at what level of authorization.
+
+Here is an example of the Identities portal of a Company, which can be found in the IAM (Identity and Access Management) section from the Company Overview, wherein it is possible to manage the Role assigned at Company level:  
+
+![Company Identities table](./img/identities_table.png)
+
+In order to manage privileges at Project level, the Identities table is shown even into Project Settings Area, wherein it is possible to manage Roles assigned to the specific Project and its Envinronments.
+
+<!-- TODO: ![Project Identities table](./img/identities_table_on_project.png) -->
+
+:::caution Important: Git provider vs Console permissions
+The management of Roles and permissions aims at regulating access on resources solely at Console level.  
+Roles and permissions of Users on the same resources at the Git provider level are not managed in this Console area; thus, Roles at Git-level must be verified in order to prevent possible discrepancies that may appear between permissions the User has at the Console-level and those the User has at the Git repository level.
+
+For example, it could happen that a User who does not even have permissions to access a Company from the Console is, however, in a Git group with an assigned Role that gives them Git-level permissions on these Company resources!
```

Comment:
```suggestion
For example, it could happen that a User who does not even have permissions to access a Company from the Console is, however, in a Git group with an assigned Role that gives them Git-level permissions on those Company resources!
```